test/cmd/txsim/cli.go

```diff
@@ -192,6 +194,7 @@ func flags() *flag.FlagSet {
 	flags.IntVar(&blob, "blob", 0, "number of blob sequences to run")
 	flags.StringVar(&blobSizes, "blob-sizes", "100-1000", "range of blob sizes to send")
 	flags.StringVar(&blobAmounts, "blob-amounts", "1", "range of blobs to send per PFB in a sequence")
+	flags.BoolVar(&useFeegrant, "feegrant", false, "use the feegrant module to pay for fees")
```

Comment:
that's fair, I think this short cli description assumes that the user is familar with the feegrant module.

its not a frequently used module afaiu, although it is actually really useful. we have a link in our specs, 

https://github.com/celestiaorg/cosmos-sdk/blob/v1.14.0-sdk-v0.46.11/x/feegrant/spec/README.md

> This module allows accounts to grant fee allowances and to use fees from their accounts. Grantees can execute any transaction without the need to maintain sufficient fees.

I'm trying to think of a line that would be better that also is short enough to keep in the cli but am having a bit of trouble. "use the feegrant module to pay for fees. Useful for users with restricted funds."

Reply:
On a separate note: I think the behavior of txsim is clearer when the `feegrant` is set to true (allowing more transactions based on the total funds in the master account, without being restricted by separate fund sections). It may be a good idea to consider a default setting of `true` for `feegrant`.